benchalerts/tests/unit_tests/expected_md/summary_noerrors_nobaselines.md

```diff
@@ -1,3 +1,5 @@
 ## Benchmarks with performance regressions
 
-Conbench could not find a baseline run for contender commit `no_basel`. A baseline run needs to be on the default branch in the same repository, with the same hardware, and have at least one of the same benchmark case/context pairs as one of the runs on the contender commit.
+There weren't enough matching historic runs in Conbench to make a call on whether there were regressions or not.
+
+To use the lookback z-score method of determining regressions, there needs to be at least one historic run on the default branch which, when compared to one of the runs on the contender commit, is in the same repository, on the same hardware, and has at least one of the same benchmark case and context pairs.
```

Comment:
> there needs to be at least one historic run

Is that true? :) Do we provide a result when we only have one datapoint in the "lookback" window?

Reply:
Good callout; switching to "at least two."